packages/ketcher-react/src/script/format/schemes/textSchema.js

```diff
@@ -0,0 +1,40 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+const schema = {
+  id: '/Text',
+  type: 'object',
+  required: ['label', 'position'],
+  properties: {
+    label: {
+      title: 'Label',
+      type: 'string'
+    },
+    position: {
+      type: 'object',
+      required: ['x', 'y'],
+      properties: {
+        x: {
+          type: 'integer'
```

Comment:
Please look at https://json-schema.org/understanding-json-schema/reference/numeric.html#:~:text=Therefore%2C%20JSON%20Schema%20can%20not,and%20not%20the%20type%20alone. to choose valid type